core/controllers/admin.py

```diff
@@ -733,14 +726,12 @@ def get(self):
         if filter_criterion == feconf.USER_FILTER_CRITERION_ROLE:
             role = self.normalized_request.get(
                 feconf.USER_FILTER_CRITERION_ROLE)
-            users_by_role = {
-                username: role
-                for username in user_services.get_usernames_by_role(role)
-            }
             role_services.log_role_query(
                 self.user_id, feconf.ROLE_ACTION_VIEW_BY_ROLE,
                 role=role)
-            self.render_json(users_by_role)
+            self.render_json({
+                'usernames': user_services.get_usernames_by_role(role)
+                })
```

Comment:
Changed the API schema as "role" is part of the query string and adding it to the response as the only value in the key-value dict doesn't look good!  Also, makes it easier to present in the frontend.

Reply:
API change sgtm, thanks.

Please deindent this line by 4 spaces to match the opening paren/brace.

Reply:
Done!